intel_extension_for_transformers/llm/runtime/graph/core/layers/mha_dense.cpp

```diff
@@ -2052,6 +2052,78 @@ void jblas_reordered_attn_fp32_shift_rope_k(char* cache, const ne_fp16_t* cossin
     }
 }
 
+template <bool zero_padding>
+void jblas_fusion_attn_fp32_batch_cpy_k_(const jblas_fusion_attn_fp32_batch_cpy_kv_args_t* params) {
+  const auto p = *params;
+  const auto pad_headsize = padto(p.head_size, 32);
+  const auto pad_seq_max = padto(p.seq_max, 48);
```

Comment:
I think 48 and 32 are from Bestla kernels, make it meaningful
```
const static N_PAD=48;
const auto pad_seq_max = padto(p.seq_max, N_PAD,)
```

Reply:
Updated in 92ff74ad17680cee6a0977f1626d86eea7bfad03